jobs/src/main/scala/dcos/metronome/repository/impl/kv/marshaller/JobRunMarshaller.scala

```diff
@@ -91,8 +91,8 @@ object JobRunConversions {
         id = proto.getId.toModel,
         jobSpec = proto.getJobSpec.toModel,
         status = proto.getStatus.toModel,
-        createdAt = new DateTime(proto.getCreatedAt, DateTimeZone.UTC),
-        completedAt = if (proto.hasFinishedAt) Some(new DateTime(proto.getFinishedAt, DateTimeZone.UTC)) else None,
+        createdAt = Instant.ofEpochMilli(proto.getCreatedAt),
+        completedAt = if (proto.hasFinishedAt) Some(Instant.ofEpochMilli(proto.getFinishedAt)) else None,
```

Comment:
I saw this construction so many times in marathon, I think I'll write a method for extracting protobuf options at some tome :)

Reply:
+1

Reply:
We have a helper method for this in Marathon

https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/api/serialization/ContainerSerializer.scala#L343